client/src/plus/deep-dives/article.tsx

```diff
@@ -0,0 +1,117 @@
+import { useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import { Feature } from "./feature";
+import { SeriesCard, SerieData } from "./series-card";
+
+import "./index.scss";
+import { DeepDiveNotFound } from "./not-found";
+import { Survey } from "./survey";
+import {
+  YourBrowserSupportToolkit,
+  PlanningForBrowserSupport,
+} from "./sample-articles";
+
+const SERIES = [
+  {
+    slug: "planning-for-browser-support",
+    title: "Planning for browser support",
+  },
+  {
+    slug: "your-browser-support-toolkit",
+    title: "Your browser support toolkit",
+  },
+];
+
+export default function Article({ slug }: { slug: string }) {
+  // Only when served via static Express does it force a trailing
+  // slash to the end of the URL. So we have to, for testing reasons,
+  // remove any trailing slashes.
+  if (slug.endsWith("/")) {
+    slug = slug.slice(0, -1);
+  }
+
+  useEffect(() => {
+    const serie = SERIES.find(
+      (serie) => serie.slug.toLowerCase() === slug.toLowerCase()
+    );
+    const title = serie ? serie.title : "Article not found";
+    document.title = title;
+  }, [slug]);
+  const { locale } = useParams();
+
+  function getSeriesList(): SerieData[] {
+    const list: SerieData[] = [];
+    for (const serie of SERIES) {
+      list.push({
+        displayName: serie.title,
+        slug: serie.slug,
+        state: serie.slug === slug.toLowerCase() ? "active" : "",
+      });
+    }
+    // Cheating a bit here manually adding one that *could be*.
+    list.push({
+      displayName: "Practical browser support",
+      slug: "#",
```

Comment:
Could we make `slug` be optional? So, if the item is not meant to be a link, you do not pass a slug?